### PR TITLE
Added linkPreviewsToCovers configuration setting.

### PR DESCRIPTION
**Pull request by demiankatz**
Currently, when the preview setting is turned on, all thumbnail links are replaced with links to third-party book previews. This is not always desirable. This pull request adds a config setting which allows preview linking to be either completely disabled, or else configured on a page-by-page basis.

### CODE REVIEW

File: themes/bootstrap3/js/preview.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ function applyPreviewUrl($link, url) {
 
     // Update associated record thumbnail, if any:
   $link.parents('.result,.record')
-        .find('.recordcover').parents('a').attr('href', url);
+        .find('.recordcover[data-linkpreview="true"]').parents('a').attr('href', url);
```
Comment (demiankatz) [inline themes/bootstrap3/js/preview.js]: My gut feeling is that it's smarter to use data-elements instead of classes when we're just trying to flag things, since it reduces the chances of unintended CSS-related side effects. If the performance impact were significant, I could of course be persuaded to use a class here, but if it's just a matter of ms, I think clarity and lack of side effects are worth the cost. On the other hand, if you think there might be a situation where somebody would WANT to style things differently based on this class, that could be another argument for using classes... except that this is just a flag class, and it doesn't tell us whether or not a preview has actually been linked, it just means that one might be. If we wanted a stylistic indication of which covers are linked to previews, we would have to apply an additional class in preview.js... so I think that's really a separate matter entirely.

So bottom line, what do you think? Do you agree that it's okay as-is, or do you lean toward the class approach over the data attribute?

Comment (crhallberg) [inline themes/bootstrap3/js/preview.js]: I guess we have no need to squabble over milliseconds. And if someone wanted to style it they could use the same selector. Fine as is.

Comment (demiankatz) [inline themes/bootstrap3/js/preview.js]: Good point regarding selectors -- nothing protects us completely against side effects. :) In any case, I'll go ahead and merge!
